### DISCUSSION TITLE
output-format 'grouped' and 'full'?

### DISCUSSION THREAD

**Discussion by kpfleming** [astral-sh/ruff#9807] [Q&A]
Version 0.2.0 deprecated `--show-source` and recommends using `--output-format=full` as a replacement. I've been using `--output-format=group --show-source`, and according to the docs `full` and `grouped` are mutually exclusive.

What should the replacement be?

**Comment by zanieb** [comment]
Hi! You can still use `--output-format=grouped --show-source` for now but we are deprecating using those together since the grouped format makes more sense when just reporting rule codes. We'll probably incorporate some sort of grouping by file into the `full` output format.

If you have some thoughts about what your ideal future display would be like or how grouping while showing sources is helpful to you please let us know!

**Reply by kpfleming** [reply]
That's fine, I'll just switch to `full` as that will be sufficient for my needs.
